app/src/ui/history/commit-list-item.tsx

```diff
@@ -219,6 +219,10 @@ export class CommitListItem extends React.PureComponent<
     clipboard.writeText(this.props.commit.sha)
   }
 
+  private onCopyTags = () => {
+    clipboard.writeText(this.props.commit.tags.join(', '))
```

Comment:
We discussed about this and think it might be more useful to use spaces instead, since it's very probable you're gonna use these copied tags in a `git command`, don't you think? 🤔

Reply:
also made the the tags in the detail view selectable